depsdotdev.go

```diff
@@ -171,6 +178,70 @@ func (c *client) GetDirectDependencies(ctx context.Context, p Package) ([]Packag
 		})
 	}
 
+	if hasBundledDependencies {
+		if p.Ecosystem == "npm" {
```

Comment:
Maybe use a string constant instead of "npm"?

Reply:
The `aggregdepscore` package is not yet declaring such constants, I think I will figure later how I want to do it.

I added a TODO comment in `[e797443](https://github.com/DataDog/aggregated-dependency-score/pull/18/commits/e79744337d73e0e23cef4850e8d882c3aece08cc)`